README.md

```diff
@@ -13,6 +13,8 @@ Please check out the following if you are new to Twilio's Programmable Voice or
 - [Programmable Voice](https://www.twilio.com/docs/voice/sdks)
 - [React Native](https://reactnative.dev/docs/getting-started)
 
+Please note that we do not officially support Expo. Our officially supported use case is a bare React Native application that targets iOS and Android. When following the React Native environment setup guide, please select "React Native CLI".
```

Comment:
This sounds like we're not open to supporting Expo in the future.  I don't think this message even belongs here. The  "please select "React Native CLI"" should be part of a setup instructions. The expo part, we don't have to mention here. Or we can add it in the common issues if it's not there yet.

Reply:
It's also part of the common issues.